code/src/OSFramework/Interface/IDataSource.ts

```diff
@@ -69,5 +69,10 @@ namespace OSFramework.Grid {
          * @param dataItem
          */
         toOSFormat(dataItem: any): any;
+        /**
+         * Transform value to JSON format
```

Comment:
By the name of the method I would not think that the method was going to transform value to JSON format :p

Reply:
Aren't you looking at the description of the next method?